ludwig/schema/utils.py

```diff
@@ -385,6 +386,116 @@ def FloatRange(
     )
 
 
+def IntegerOrSequenceOfIntegers(
+    default: Union[None, int, Tuple[int, ...], TList[int]] = None, allow_none=False, description=""
+):
+    """Returns a dataclass field with marshmallow metadata enforcing numeric inputs or a tuple of numeric
+    inputs."""
+    val = validate.OneOf([int, tuple, list])
+    allow_none = allow_none or default is None
+
+    if default is not None:
+        try:
+            assert isinstance(default, int) or isinstance(default, tuple) or isinstance(default, list)
+            val(type(default))
+            if isinstance(default, tuple) or isinstance(default, list):
+                for i in default:
+                    assert isinstance(i, int)
+        except Exception:
+            raise ValidationError(f"Invalid default: `{default}`")
+    return field(
+        metadata={
+            "marshmallow_field": fields.Integer(
```

Comment:
Does this work as intended? Prima facie it would seem this only works for the `int` case but not `tuple` or `list` (